homeassistant/components/isy994/__init__.py

```diff
@@ -17,7 +17,8 @@
 from homeassistant.helpers.entity import Entity
 from homeassistant.helpers.typing import ConfigType, Dict
 
-REQUIREMENTS = ['PyISY==1.1.1']
+REQUIREMENTS = [
+    'https://github.com/shbatm/PyISY/archive/develop.zip#PyISY==1.1.2']
```

Comment:
Only PyPi published packages are allowed

Reply:
Just an update, I am still working on incorporating the changes requested above; however, I am also still waiting on the upstream package to be updated--this requirement will be reverted to the PyPi package once updated.  See note in original pull request description.